app/controllers/posts_controller.rb

```diff
@@ -6,8 +6,38 @@ def index
     @posts = @user.posts
   end
 
+  def new
+    @post = Post.new
+    @current_user = ApplicationController.current_user
+  end
+
+  def create
```

Comment:
- [ ] Based on the requirements of this project, you'll require a pair of routes for creating posts ie. posts#new, posts#create hence do not expose unnecessary routes in the routes.rb file.
- Hints: `resources :posts, only: [:new, :create] do` The rest consumed up our memory space.

Reply:
- Well resolved 👍🏽